app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -1,5 +1,4 @@
-$govuk-cookie-banner-background: #d5e8f3;
-$govuk-cookie-banner-background-new: govuk-colour("white");
+$govuk-cookie-banner-background: govuk-colour("white");
 $govuk-cookie-banner-text-green: #00823b;
```

Comment:
Probably outside the scope of this PR, but does this need to be a variable - it looks like it's only used once?

Reply:
@andysellick I think this is more to appease the linter